Apps/Inc/main.c

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "BSP_CAN.h"
+#include "config.h"
+// #include "ALEX FOLDER " ==> will hold checkVoltage() function
+
+int main(void){
+    while(checkVolatge() > 0){
+        BSP_CAN_Write(0x10B, checkVoltage() )     //send message to CAN Controller 
```

Comment:
There should be a semicolon here. Also this file should be in Apps/Src so replace the main.c file that is in Src and place this in there instead.

Reply:
Also, the data ID for voltage data is 0x104